src/main/scala/com/amazon/deequ/VerificationSuite.scala

```diff
@@ -116,7 +116,7 @@ class VerificationSuite {
         VerificationFileOutputOptions())
     : VerificationResult = {
 
-    val analyzers = requiredAnalyzers ++ checks.flatMap { _.requiredAnalyzers() }
+    val analyzers = (requiredAnalyzers ++ checks.flatMap { _.requiredAnalyzers() }).distinct
```

Comment:
this should not be necessary

Reply:
So I added this because it was failing multiple checks and one of the main scenarios are:

If requiredAnalyzers is passed with checks during VerificationSuite then it will throw error because it adds analyzers 2 times (1 from analyzers itself and 2 from checks requiredAnalyzers) 
https://github.com/awslabs/deequ/blob/master/src/test/scala/com/amazon/deequ/VerificationResultTest.scala#L184

That's why I took distinct

And also it fails for:
https://github.com/awslabs/deequ/blob/master/src/test/scala/com/amazon/deequ/VerificationSuiteTest.scala#L251

Where multiple anomalyChecks used for same analyzer where Size analyzers is taken multiple times

Reply:
What if we add the difference between check analyzer and normal analyzer during verificationRun ... This will solve that use case and give error when user specify duplicate analyzers during this run

Reply:
I think we should fix the test to not configure duplicate analyzers and leave the code as is. Let us not mangle with the analyzers configured by the users, lets just notify them in case of duplicates.

Reply:
So are you suggesting me to fix all those test cases which were failing ? And remove this line with distinct ?

Also is there any problem using it like:
`val analyzers = requiredAnalyzers ++ checks.flatMap { _.requiredAnalyzers() }.diff(requiredAnalyzers)`

If above code snippet works then it will solve everything else I'll fix the test cases

Reply:
Please fix the test cases and do not modify the analyzers specified by the users.

Reply:
@sscdotopen I have made necessary changes to test cases and reverted back this code line 

Now all test cases is passing, I have not added/deleted any analyzers but removed those which were added multiple times and causing duplicates error can you please check and let me know if it looks good